action/protocol/execution/protocol_test.go

```diff
@@ -457,12 +457,18 @@ func TestProtocol_Handle(t *testing.T) {
 			delete(cfg.Plugins, config.GatewayPlugin)
 		}()
 
-		testTrieFile, _ := ioutil.TempFile(os.TempDir(), "trie")
+		testTrieFile, err := ioutil.TempFile(os.TempDir(), "trie")
+		require.NoError(err)
 		testTriePath := testTrieFile.Name()
-		testDBFile, _ := ioutil.TempFile(os.TempDir(), "db")
+		require.NoError(testTrieFile.Close())
```

Comment:
the file is closed here, but 
sf, err := factory.NewStateDB(cfg, factory.DefaultStateDBOption())
then it will be opened again in
require.NoError(bc.Start(ctx))